addons/website/views/website_navbar_templates.xml

```diff
@@ -20,6 +20,11 @@
             <t t-set="body_classname" t-value="(body_classname if body_classname else '') + (' o_connected_user' if env['ir.ui.view'].user_has_groups('base.group_user') else '')"/>
         </xpath>
         <xpath expr="//div[@id='wrapwrap']" position="before">
+            <t t-if="request.httprequest.args.get('with_loader') == '1'">
```

Comment:
`<div  t-if="'with_loader' in request.params"> class="o_theme_install_loader_container`